src/Accounts/Accounts/help/Update-AzConfig.md

```diff
@@ -277,6 +278,36 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -EnableTestCoverage
```

Comment:
Please remove unrelated parameters
Please talk to Vincent about how we can avoid generating this